debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/elf/ELFDumpReader.java

```diff
@@ -1210,8 +1210,10 @@ public void walkStack() {
 				instructionPointer = maskInstructionPointer(getLinkRegisterFrom(registers));
 			}
 
-			if (0 != instructionPointer && 0 != basePointer	&& isValidAddress(instructionPointer)
```

Comment:
The basePointer check may be important for 32-bit code.

Reply:
The check for `0 != basePointer` is moved down before the value is actually used. I assume retrieving stack frames via `unwindTable` works regardless the value of `basePointer`. Also verified that `xi3280 jdmpview !nstack` shows native stacks, and passes `VM_Extended.testDTFJ` test.